odkx-src/basics-install.rst

```diff
@@ -8,15 +8,10 @@ These instructions describe the steps to install the ODK-X basic tools on a tabl
 Prerequisites
 -------------------
 
-You must have an Android tablet with an operating system version 4.4 or higher.
-
-If you are working on a Windows/Mac/Linux machine, you can use `Android Studio <https://developer.android.com/studio>`_ to launch an Android emulator for testing purposes.
-
-Please note that ODK-X Services version 2.1.7 doesn't work on Android 11. You will need Android 10 with API level less than 30 for version 2.1.7.
-
-Before installing any of the ODK-X tools, you will need the following third party app:
-
-  - `OI File Manager <https://github.com/openintents/filemanager/releases>`_
+- An Android tablet with an operating system version 4.4 or higher. If you are working on a Windows/Mac/Linux machine, you can use `Android Studio <https://developer.android.com/studio>`_ to launch an Android emulator for testing purposes.
+- Android 10 with API level less than 30 for version 2.1.7.
```

Comment:
This makes it sound like you have to have Android 10, that this is a pre-requisite. 

I think the original text in this section was clearer, as it was complete sentences (the first pre-requisite in the revised is a fragment) and made clear the issue with Android 10.